scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/MinionProcessor.java

```diff
@@ -0,0 +1,30 @@
+package ca.on.oicr.gsi.runscanner.scanner.processor;
+
+import ca.on.oicr.gsi.runscanner.dto.NanoporeNotificationDto;
+import ch.systemsx.cisd.hdf5.IHDF5StringReader;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class MinionProcessor extends BaseNanoporeProcessor {
+  public MinionProcessor(Builder builder, String seqName) {
+    super(builder, seqName);
+  }
+
+  @Override
+  protected Stream<Path> readsDirectoryForRun(Path path) {
+    return IntStream.of(0).mapToObj(i -> path.resolve(Paths.get("fast5", Integer.toString(i))));
```

Comment:
If this can only ever be one thing, just return `Stream.of(path.resolve(Paths.get("fast5", "0"))`